src/schemas/hierarchical-sets.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/hubmapconsortium/vitessce/#hierarchical-sets",
+    "title": "Vitessce hierarchical sets data",
+    "type": "object",
+    "definitions": {
+      "stringArray": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "colorArray": {
+          "type": "array",
+          "items": { "type": "number", "minimum": 0, "maximum": 255 },
+          "minItems": 3,
+          "maxItems": 3
+      }
+    },
+    "additionalProperties": false,
+    "required": ["datasetId", "setsType", "timestamp", "setsTree"],
+    "properties": {
+      "datasetId": { "type": "string" },
+      "setsType": { "type": "string" },
+      "timestamp": { "type": "string" },
+      "setsTree": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["key", "name"],
+            "properties": {
+              "key": { "type": "string" },
+              "name": { "type": "string" },
+              "color": { "$ref": "#/definitions/colorArray" },
+              "set": { "$ref": "#/definitions/stringArray" }
```

Comment:
- It would be one less conditional if everything had a color assigned automatically... Is there a usecase for sets without assigned colors?
- Should `set` be required?

Reply:
- The constructor for the Node class assigns default color if none is provided. 
- `set` should not be required. For example the node `Factors` right now is just a dummy node that does not have its own set.